azure/aks-full-sample/src/main/java/com/yanchware/fractal/azure/aks/sample/components/AksComponent.java

```diff
@@ -29,8 +28,8 @@ public static AzureKubernetesService getAks(String id) {
         .withVnetAddressSpaceIpRange("10.1.0.0/22")
         .withVnetSubnetAddressIpRange("10.1.0.0/22")
         .withExternalWorkspaceResourceId("workplaceResourceId")
-        .withExternalLoadBalancerOutboundIps(List.of("ip1", "ip2"))
-        .withAddonProfiles(List.of(AzureAddonProfile.AZURE_POLICY, AzureAddonProfile.AZURE_KEYVAULT_SECRETS_PROVIDER))
+        .withOutboundIps(getOutboundIps())
```

Comment:
You need to add kubernetes version here to show the usage of it, it is a "full example". Look at my PR for the changes.

Reply:
sure